src/QuizContent.js

```diff
@@ -0,0 +1,204 @@
+                        </Grid>
+                    )) : <div> </div>
+                }
+                </Grid>
+            </RadioGroup>
+            <div style ={{
+                display: 'flex', 
+                flexDirection: 'row',
+                justifyContent: 'space-between' 
+            }}>
+                <div>
+                    <Button variant = "outlined" onClick={() => onBackQuestion(resetSelection)} sx={{mt:5, ml:1}}>
+                        Previous Question
+                    </Button>
+                    <Button variant = "outlined" onClick={() => onSkipQuestion(resetSelection)} sx={{mt:5, ml:1}}>
+                        Next Question
+                    </Button>
+                </div>
+
+                <Button variant = "contained" onClick={() => onAnswerSelected(selected, resetSelection)} sx={{m:0, mt:5}}>
+                    Submit Answer
+                </Button>
+            </div>
+        </Container>
+    );
+}
+
+export default function QuizContent () {
+
+    const [questions, setQuestions] = useState([]);
+    const [quizTitle, setQuizTitle] = useState("");
+    const [currentQuestionIndex, setCurrentQuestionIndex] = useState(0);
+
+    const fetchQuizData = async () => {
+        const options = {
+            method: "GET",
+            headers: 
+                {Authorization: `Bearer ${null}`}
+        };
+
+        const url = ''
+
+        try {
+            const response = await fetch(url, options)
+            if(!response.ok){
+                console.log(response);
+                const errMessage = `${response.status}`
+                throw new Error(errMessage);
+            }
+
+            const quizData = await response.json();
+
+            return quizData;
+        } catch (error) {
+            console.log(error.message);
+        }
+    }
+
+   useEffect(()=> {
+   //    const fetchQuiz = async function () {
+    //        try {
+    //            const quizQs = await fetchQuizData();
+               
+    //            const fetchedQuestions = await fetchQuizData();
+    
+    //            setQuestions(fetchedQuestions);
+    
+    //        } catch (error) {
+    //            console.error(error.message);
+    //        }
+    //    }
+        const title = "Addition Quiz"    
+
+        const quizQs = [            
+            { id: 1, question: "2+2 equals:", options: [3, 4, 5, 6], correctAnswer: 4, userAnswer: null, isCurrentQ:null },
+            { id: 2, question: "2+3 equals:", options: [3, 4, 5, 6], correctAnswer: 5, userAnswer: null, isCurrentQ:null},
+            { id: 3, question: "2+4 equals:", options: [3, 4, 5, 6], correctAnswer: 6, userAnswer: null, isCurrentQ:null},
+            { id: 4, question: "2+1 equals:", options: [3, 4, 5, 6], correctAnswer: 3, userAnswer: null, isCurrentQ:null}
+        ];
+
+        setQuestions(quizQs);
+        setQuizTitle(title);
+    }, [])
+
+    const handleAnswerSelected = (answer, resetCallback) => {
```

Comment:
`handleAnswerSelected` has no parameters but 2 arguments passed

Reply:
Hey Sergey, the arguments are being passed inside the QuestionContent component.  The function is just being passed to the component as a prop so you aren't seeing the parameters.